src/content/contributing/adding-layer-2/index.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Adding Layer 2s
+description: The policy we use when adding a layer 2 to ethereum.org
+lang: en
+sidebar: true
+---
+
+# Adding layer 2s {#adding-layer-2}
+
+We want to make sure we list the best resources possible so users can navigate the layer 2 space in a safe and confident manner.
+
+Anyone is free to suggest adding a layer 2 on ethereum.org. If there's a layer 2 that we have missed, **please suggest it!**
+
+We currently list L2s on the following pages:
+
+- [Optimistic rollups](/developers/docs/scaling/optimistic-rollups/)
+- [Zero-knowledge rollups](/developers/docs/scaling/zk-rollups/)
+<!-- TODO: Add layer-2 page -->
+
+## How we decide {#how-we-decide}
+
+Layer 2 submissions will be assessed by the following criteria:
+
+**[Listing on L2BEAT](https://l2beat.com)**
+
+- In order to be considered, this project to be featured on L2BEAT. This resource provides a great risk assessment of layer 2 projects. **If the project is not featured on L2BEAT, they will not be considered for ethereum.org.**
```

Comment:
Feel free to edit but the unasked question I'm trying to answer is: why would ethereum.org trust L2BEAT's analysis instead of doing it themselves?

```suggestion
- In order to be considered, this project must be featured on L2BEAT. L2BEAT provides a great risk assessment of layer 2 projects that we lean on for evaluating L2 projects. **If the project is not featured on L2BEAT, we will not list them as an L2 on ethereum.org.**
```